frontend/pages/Homepage/cards/Software/Software.tsx

```diff
@@ -54,104 +51,50 @@ const EmptySoftware = (message: string): JSX.Element => {
-    lastUpdatedAt = "never";
-  } else {
-    lastUpdatedAt = formatDistanceToNowStrict(new Date(lastUpdatedAt), {
-      addSuffix: true,
-    });
-  }
-  return (
-    <span className="last-updated">
-      {`Last updated ${lastUpdatedAt}`}
-      <span className={`tooltip`}>
-        <span
-          className={`tooltip__tooltip-icon`}
-          data-tip
-          data-for="last-updated-tooltip"
-          data-tip-disable={false}
-        >
-          <img alt="question icon" src={QuestionIcon} />
-        </span>
-        <ReactTooltip
-          place="top"
-          type="dark"
-          effect="solid"
-          backgroundColor="#3e4771"
-          id="last-updated-tooltip"
-          data-html
-        >
-          <span className={`tooltip__tooltip-text`}>
-            Fleet periodically
-            <br />
-            queries all hosts
-            <br />
-            to retrieve software
-          </span>
-        </ReactTooltip>
-      </span>
-    </span>
-  );
-};
-
 const Software = ({
   currentTeamId,
   setShowSoftwareUI,
   showSoftwareUI,
   setActionLink,
   setTitleDetail,
 }: ISoftwareCardProps): JSX.Element => {
-  const [isLoadingSoftware, setIsLoadingSoftware] = useState<boolean>(true);
   const [navTabIndex, setNavTabIndex] = useState<number>(0);
   const [pageIndex, setPageIndex] = useState<number>(0);
 
-  const { data: software, error: errorSoftware } = useQuery<
-    ISoftwareResponse,
-    Error
-  >(
+  const {
+    data: software,
+    isFetching: isSoftwareFetching,
+    error: errorSoftware,
+  } = useQuery<ISoftwareResponse, Error>(
     [
       "software",
       {
         pageIndex,
         pageSize: PAGE_SIZE,
-        // searchQuery,
         sortDirection: DEFAULT_SORT_DIRECTION,
         sortHeader: DEFAULT_SORT_HEADER,
         teamId: currentTeamId,
         vulnerable: !!navTabIndex, // we can take the tab index as a boolean to represent the vulnerable flag :)
       },
     ],
-    () => {
-      setIsLoadingSoftware(true);
-      return softwareAPI.load({
+    () =>
+      softwareAPI.load({
         page: pageIndex,
         perPage: PAGE_SIZE,
-        // query: searchQuery,
         // TODO confirm sort is working?
         orderKey: DEFAULT_SORT_HEADER,
         orderDir: DEFAULT_SORT_DIRECTION,
         vulnerable: !!navTabIndex, // we can take the tab index as a boolean to represent the vulnerable flag :)
         teamId: currentTeamId,
-      });
-    },
+      }),
     {
-      // initialData: { software: [], counts_updated_at: "" },
-      // placeholderData: { software: [], counts_updated_at: "" },
-      // enabled: true,
-      // If keepPreviousData is enabled,
-      // useQuery no longer returns isLoading when making new calls after load
-      // So we manage our own load states
       keepPreviousData: true,
-      staleTime: 30000, // TODO: Discuss a reasonable staleTime given that counts are only updated infrequently?
```

Comment:
@martavis, just checking on the deletion of the staleTime. I think it made for smooth pagination back to previous pages if we used something like 30 seconds like we did here. Did you see any issues with it when you were testing?

Reply:
Oh I see. I removed it because we are now reloading the data every time we switch between platforms. I didn't realize it was there for pagination, so I will add it back.